fe/projects/Seller.Portal/src/shared/components/Catalog/Catalog.js

```diff
@@ -177,27 +178,201 @@ function Catalog(props) {
             });
     };
 
+    const handleChangeEntityOrder = (item) => {
```

Comment:
**suggestion:** Instead of passing the entire object, would it be worth passing only the id and order? Example:

```js
const handleChangeEntityOrder = (id, order) => {
        dispatch({ type: "SET_IS_LOADING", payload: true });

        const requestPayload = {
            id
            order
        };

        const requestOptions = {
            method: "POST",
            headers: { "Content-Type": "application/json", "X-Requested-With": "XMLHttpRequest" },
            body: JSON.stringify(updateParameters)
        };

        const url = props.updateOrderUrl + "/Order";

        return fetch(url, requestOptions)
            .then(function (response) {
                dispatch({ type: "SET_IS_LOADING", payload: false })

                return response.json().then((jsonResponse) => {

                    if (!response.ok) {
                        toast.error(jsonResponse.message);
                    }
                });
            })
    };
```